tests/network/expose.go

```diff
@@ -48,16 +47,6 @@ func newLabeledVMI(label string, virtClient kubecli.KubevirtClient, createVMI bo
 		libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding(ports...)),
 		libvmi.WithNetwork(v1.DefaultPodNetwork()))
 	vmi.Labels = map[string]string{"expose": label}
-
-	var err error
-	if createVMI {
-		vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
-		Expect(err).ToNot(HaveOccurred())
-		tests.WaitForSuccessfulVMIStartIgnoreWarnings(vmi)
```

Comment:
`tests.RunVMIAndExpectLaunch` is not ignoring warnings.
IDK why `IgnoreWarnings` was used in the first place and maybe it indeed not needed.
It would be great if you explain in the commit message why `IgnoreWarnings` is not needed any more,

Reply:
This replaced code uses the equivalent of tests.RunVMIAndExpectLaunchIgnoreWarnings() but I believe that this is only because it is a remnant of the 2018 commit fc0d824da which introduced expose tests. Since then KubeVirt has gone a long way, so I assume the warnings are gone. Repeated testing in CI have not exposed failure in the modified code path.

If I am mistaken, the code can reintroduce tests.RunVMIAndExpectLaunchIgnoreWarnings() very quickly. On the other hand, if you request, I would do so right now.